test/TestServer.js

```diff
@@ -0,0 +1,50 @@
+'use strict';
+
+const P = require('bluebird');
+const ServiceRunner = require('service-runner');
```

Comment:
Did you mean(TM) `require('../service-runner.js')` ? As written, this might have unexpected consequences if, e.g., somebody has `service-runner` installed globally.

Reply:
Done